src/routes/e2e/widget/modal/+page.svelte

```diff
@@ -22,23 +20,29 @@
     userResponse = null;
   }
 
-  componentEvents.subscribe((event) => {
+  componentEvents.subscribe((event: { lastAction: string; reason: any }) => {
     if (event.lastAction === 'mount') {
       console.log('Modal mounted');
     }
     if (event.lastAction === 'close') {
       console.log(`Modal closed due to ${event && event.reason}`);
     }
   });
-  journeyEvents.subscribe((event) => {
-    if (event?.user?.successful) {
-      console.log(event.user);
-      userResponse = event.user;
-    }
-    if (event.journey.error || event.oauth.error || event.user.error) {
-      console.log('Login failure event fired');
-    }
-  });
+  journeyEvents.subscribe(
+    (event: {
+      user: { successful: any; error: any };
+      journey: { error: any };
+      oauth: { error: any };
+    }) => {
```

Comment:
I feel like we shouldn't have to define these types. They should be automatically typed, or, at the very least, we should be able to import them in, yeah?

Reply:
Were you not able to remove them?